charts/opentelemetry-ebpf/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v2
+name: opentelemetry-ebpf
+version: 0.1.0
+description: OpenTelemetry eBPF Helm chart for Kubernetes
+type: application
+home: https://opentelemetry.io/
+sources:
+  - https://github.com/open-telemetry/opentelemetry-collector
+  - https://github.com/open-telemetry/opentelemetry-collector-contrib
```

Comment:
Are these actually sources for this chart?

Reply:
I think it could be and only  `https://github.com/open-telemetry/opentelemetry-ebpf` enough